deploy/docker/.env

```diff
@@ -1,4 +1,51 @@
-DINKY_VERSION=0.8.0
-FLINK_HOME=opt/flink_home
-FLINK_BIG_VERSION=1.16
-FLINK_VERSION=1.16.1
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+DINKY_VERSION=1.0.3
+FLINK_VERSION=1.17
+
+# Customize other dependent paths,
+# which will be mounted to dinky and flink containers
+# 自定义其他依赖路径，会挂载到dinky和flink容器中
+CUSTOM_JAR_PATH=/opt/dinky/extends/
+FLINK_PROPERTIES="jobmanager.rpc.address: jobmanager"
+TZ=Asia/Shanghai
+
+# mysql,h2,pgsql
+DB_ACTIVE=h2
+
+# use h2 db localtion file path
+H2_DB=./tmp/db/h2
```

Comment:
will conflict with flyaway?

Reply:
> will conflict with flyaway?

only in docker, if the user wants to upgrade the dinky version, the container must be re-created and the corresponding h2 file will disappear, so there is no flyaway problem